frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetOverview/ClusterSetOverview.tsx

```diff
@@ -52,87 +79,108 @@ export function ClusterSetOverviewPageContent() {
                         {
                             key: t('table.clusterSetBinding'),
                             keyAction: (
-                                <Popover
-                                    bodyContent={
-                                        <Trans
-                                            i18nKey="clusterSetBinding.edit.message"
-                                            components={{ bold: <strong /> }}
-                                        />
-                                    }
-                                >
-                                    <AcmButton variant="link" style={{ padding: 0, paddingLeft: '6px' }}>
-                                        <OutlinedQuestionCircleIcon />
+                                <Fragment>
+                                    <Popover
+                                        bodyContent={
+                                            <Trans
+                                                i18nKey="clusterSetBinding.edit.message"
+                                                components={{ bold: <strong /> }}
+                                            />
+                                        }
+                                    >
+                                        <AcmButton variant="link" style={{ padding: 0, paddingLeft: '6px' }}>
+                                            <OutlinedQuestionCircleIcon />
+                                        </AcmButton>
+                                    </Popover>
+                                    <AcmButton
+                                        onClick={() => {
+                                            setShowManagedClusterSetBindingModal(true)
+                                        }}
+                                        variant="link"
+                                        style={{ padding: 0, paddingLeft: '6px' }}
+                                    >
+                                        <PencilAltIcon />
                                     </AcmButton>
-                                </Popover>
+                                </Fragment>
                             ),
                             value: clusterSetBindings?.length ? (
                                 <AcmLabels labels={clusterSetBindings?.map((mcsb) => mcsb.metadata.namespace!)} />
                             ) : (
                                 '-'
                             ),
                         },
+                        {
+                            key: t('table.userManagement'),
+                            value: `Users ${users ? users : '-'} Groups ${groups ? groups : '-'}`,
```

Comment:
Please externalize with interpolation variables. I would suggest using 2 separate strings, so that they can be independently plural or singular, joined by ", ". (See translation.json keys with suffix `_plural` and interpolation variable `count` for an example.)

Both 0: "None"
1 user, 0 groups: "1 user"
2 users, 1 group: "2 users, 1 group"

Reply:
Done, Thanks